xtask/src/build/jco.rs

```diff
@@ -14,6 +13,17 @@ pub(crate) fn run() -> Result<()> {
         "wasm-tools".to_string(),
     )?;
 
+    let sh = Shell::new()?;
+    //Typescript doesn't support arbitrary module namespace identifier names so
+    //we need to remove it from wasm-tools.js before compiling
+    //See: https://github.com/microsoft/TypeScript/issues/40594
+    cmd!(
+        sh,
+        "sed -i'' -e 's/tools as \"local:wasm-tools\\/tools\"//g' ./obj/wasm-tools.js"
```

Comment:
Rather than doing this via replacement (since if jco has this issue itself then other libraries built with jco will too), perhaps we should instead make it a transpile option to omit these names?

Perhaps something like `--ts-compat` or `--no-namespaced-exports`. Would you be interested in working on a patch along those lines here?

Reply:
Yep happy to take a look at that (and fix this windows build issue). Will probably take me a few more days or through the weekend, but any pointers to that section of the code (and potentially how cli options flow from the js to the rust bits) would be appreciated

Reply:
Sure, the exports are output by the "esm bindgen" render routine in https://github.com/bytecodealliance/jco/blob/main/crates/js-component-bindgen/src/esm_bindgen.rs#L184.

Reply:
Alright updates are made and the `--no-namespaced-exports` option has been added